tools/lint-md/lint-md.src.mjs

```diff
@@ -35,16 +48,36 @@ paths.forEach(async (path) => {
   const fileContents = file.toString();
   const result = await linter.process(file);
   const isDifferent = fileContents !== result.toString();
+  let manPageContent;
+  let isManPageModified = false;
+  if (resolve(path) === cliPath) {
+    try {
+      child_process.execFileSync(process.execPath, [
+        npxPath,
+        '--yes',
+        'github:nodejs/api-docs-tooling',
```

Comment:
Using npx sounds like very bad DX, we will end up with different versions on different machines, won't we? I'm -1 on that, let's do like we do for linting the version number: the linter may or may not get the info from the env, if not it doesn't check it, and the CI can still report problems, and the devs stay happy: https://github.com/nodejs/node/blob/8de2695fe5e4eb4fd4bafc24e9d94fbe2df4ae86/.github/workflows/linters.yml#L107-L116

Reply:
If/when the PR to stop pre-compiling lint-md lands, this can be converted to not use npx.